dtmi/sgs/ocm_online-2.json

```diff
@@ -0,0 +1,275 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:SGS:OCM_Online;2",
+  "@type": "Interface",
+  "displayName": "OCM-Online",
+  "description": "Report the current oil quality data and oil iron content distribution data, and Get Log for STAS-PLUS",
+  "contents": [
+    {
+      "@type": [
+        "Property",
+        "Grade"
+      ],
+      "name": "Grade4um",
+      "displayName": ">4um Grade",
+      "description": "Particle grade greater than 4um",
+      "schema": "integer",
+      "unit": "Grade"
```

Comment:
"Grade" is not really a unit, is it?  What does the integer number of this Property mean?

Reply:
I modified it